src/libslic3r/GCode/ToolOrdering.hpp

```diff
@@ -150,11 +150,11 @@ class ToolOrdering
 
     // For the use case when each object is printed separately
     // (print->config().print_sequence == PrintSequence::ByObject is true).
-    ToolOrdering(const PrintObject &object, unsigned int first_extruder, bool prime_multi_material = false);
+    ToolOrdering(const PrintObject &object, unsigned int first_extruder, bool is_BBL_printer, bool prime_multi_material = false);
 
     // For the use case when all objects are printed at once.
     // (print->config().print_sequence == PrintSequence::ByObject is false).
-    ToolOrdering(const Print& print, unsigned int first_extruder, bool prime_multi_material = false);
+    ToolOrdering(const Print& print, unsigned int first_extruder, bool is_BBL_printer, bool prime_multi_material = false);
```

Comment:
We don't need to pass `bool is_BBL_printer`, we can query it by calling `print.is_BBL_printer()`